.jenkins/staticanalysis.groovy

```diff
@@ -11,7 +11,7 @@ def runCompileCommand(platform, project, jobName, boolean debug=false)
     def command = """#!/usr/bin/env bash
             set -ex
             cd ${project.paths.project_build_prefix}
-            doxygen docs/Doxyfile
+            doxygen doc/Doxyfile
```

Comment:
I would rename the folder to docs. All other libraries it is called docs. Just some standardization to make life easier.